src/js/vre/chapter36/config/form.js

```diff
@@ -258,6 +289,226 @@ const formConfig = {
     additionalInformation: {
       title: 'Additional Information',
       pages: {
+        additionalInformationVeteran: {
+          title: 'Additional Information',
+          path: 'additional-information',
+          depends: {
+            'view:isVeteran': true
+          },
+          uiSchema: {
+            previousBenefitApplications: previousBenefitApplicationsUI
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              previousBenefitApplications: {
+                type: 'object',
+                properties: {
+                  chapter31: {
+                    type: 'boolean'
+                  },
+                  ownServiceBenefits: {
+                    type: 'boolean'
+                  },
+                  other: {
+                    type: 'boolean'
+                  },
+                  none: {
+                    type: 'boolean'
+                  }
+                }
+              }
+            }
+          }
+        },
+        additionalInformationCurrentSpouse: {
+          title: 'Additional Information',
+          path: 'additional-information-spouse',
+          depends: {
+            applicantRelationshipToVeteran: 'Spouse'
+          },
+          uiSchema: {
+            divorceOrAnnulmentPending: {
+              'ui:title': 'If you are the spouse of a disabled Veteran is a divorce or annulment pending?',
+              'ui:widget': 'yesNo'
+            },
+            previousBenefitApplications: _.merge(
+              _.unset(
+                ['chapter31', 'ownServiceBenefits'],
+                previousBenefitApplicationsUI
+              ),
+              {
+                dic: {
+                  'ui:title': benefitsLabels.dic
+                },
+                other: {
+                  'ui:title': benefitsLabels.other
+                },
+                none: {
+                  'ui:title': benefitsLabels.none
+                }
+              }
+            ),
+            previousVeteranBenefitsFullName: _.merge(fullNameUI, {
+              'ui:description': "Veteran's name under whom you've claimed benefits",
+
+              first: {
+                'ui:required': () => false
```

Comment:
I don't think you need these if you're unset the required property in the regular schema.

Reply:
Thanks, removed.